src/pages/settings/PasswordPage.js

```diff
@@ -86,8 +80,8 @@ class PasswordPage extends Component {
         }
     }
 
-    isValidPassword() {
-        return this.state.newPassword.match(CONST.PASSWORD_COMPLEXITY_REGEX_STRING);
+    isInvalidPassword() {
```

Comment:
Why change from `isValidPassword` to `isInvalidPassword`? We usually use the "positive" version of things rather than check if something is "not" something like `isNotValid()` or `isInvalid()`.

Reply:
Okay, earlier this function was used for a flag that we don’t show. I’ll update the logic to use ‘isValidPassword’

Reply:
@marcaaron Are you suggesting we have 

```
isValidPassword() {
        return this.state.newPassword.match(CONST.PASSWORD_COMPLEXITY_REGEX_STRING);
}

and then `this.state.newPassword && !isValidPassword()`
```

That is what we initially thought to use but wanted to move this && to a separate function. Let me know and I'll accordingly update.

Reply:
Good point on keeping the function positive, in that case, yes I think we'd have to leave that logic in the render function the way you specified (maybe split out the style array into separate lines like we do [here](https://github.com/Expensify/App/blob/998d25e660d941fa2c1dbe8cff5f87e242337614/src/components/Button.js#L122) to avoid a giant line).

Alternatively, you could change the function to be called something like `shouldApplyPasswordErrorStyles` and include the empty state check, but I don't like that solution as much.

Reply:
Yeah agree and hence, lets just split into lines with an `&&` check

Reply:
I don't know why we need two separate lines. If there's no password it's not valid, right?

```js
return this.state.newPassword && this.state.newPassword.match(CONST.PASSWORD_COMPLEXITY_REGEX_STRING);
```

Reply:
Or maybe we just mean syntactically use a line break? That seems fine.

Reply:
We wanted to match the same behavior from the screenshots in https://github.com/Expensify/App/pull/5093 where the field shows as grey initially if the user hasn't typed anything.

Reply:
Changed to `isValidPassword`